docs/getting-started/server/database/ef/index.md

```diff
@@ -16,16 +16,17 @@ Our EF implementations currently support Postgres and mySQL.
 
 ## Setting up EF databases
 
-The workflow here is broadly the same as with the normal MSSQL implementation: set up the docker
-container, configure user secrets, and run the scripts in the `scripts` folders against their
-relating databases in chronological order.
+The workflow here is broadly the same as with the normal MSSQL implementation - you begin by setting
+up the Docker container and configure user secrets. However, unlike with MSSQL, we then use the
+`migrate.ps1` script to run `dotnet ef` commands to set up the database and perform the migrations
```

Comment:
This is actually identical. Remember, this is a dev instance, not production. Devs have to run migraitons manually. The only difference is in which flag is added to the migration script.

Reply:
Ah, I should have been clearer.  What I was trying to say was that the `migrate.ps1` script runs `dotnet ef` commands for the EF migrations, whereas it just runs the `DbScripts` for the MSSQL migrations.  I was trying to provide some insight into what `migrate.ps1` does under the covers.